guidance_note_wb.md

```diff
@@ -7,102 +7,149 @@ title: "Guidance to Authors"
 
 ## Reproducibility package submission
 
-To initiate the process, the authors complete a short reproducibility verification request form available [here](https://survey.wb.surveycto.com/collect/prwp_reproducibility?caseid=).
-Authors provide the reproducibility package through a GitHub repository, a OneDrive folder or other cloud-based sharing service, or, for small datasets, a compressed folder sent directly by email to [reproducibility@worldbank.org](mailto:reproducibility@worldbank.org).
-The reproducibility package must contain all elements in the [Reproducibility Package Checklist](https://github.com/worldbank/wb-reproducible-research-repository/blob/main/reproducibility_package_checklist.md).
-The package should be organized in a file structure that facilitates re-running the code (e.g., data, code, and outputs are in separate folders and the package includes [README](https://github.com/worldbank/wb-reproducible-research-repository/raw/main/resources/README_Template.docx) and ideally a [main script](https://github.com/worldbank/wb-reproducible-research-repository/blob/main/resources/main.do) that clearly specifies the order in which code files should be run).
+To begin, authors must complete the [Reproducibility Verification Request Form](https://survey.wb.surveycto.com/collect/prwp_reproducibility?caseid=) and submit their package through a GitHub repository, a OneDrive folder, another cloud-based sharing service, or, for small datasets, a compressed folder sent directly by email to reproducibility@worldbank.org. 
+
+The package should follow the [Reproducibility Package Checklist](https://github.com/worldbank/wb-reproducible-research-repository/blob/main/reproducibility_package_checklist.md). 
+It needs to clearly document the data, code, and steps used to generate the paper’s results. 
+A well-structured package typically organizes data, code, and outputs in separate folders, includes a README file, and provides a main script that specifies the order in which the code should be run. 
+Supporting resources can be found [here](https://worldbank.github.io/wb-reproducible-research-repository/resources.html).
+
+### Documenting Data  
+
+A reproducibility package should always begin from documented data sources, often referred to as *source data*. Source data is a citable dataset that is properly documented, either publicly accessible or archived in a World Bank internal catalog, and serves as the starting point for reproducibility.
+
+For each external dataset used, authors are expected to provide a full data citation. This should include a URL for publicly available datasets, or clear instructions on how to compile or request access for datasets that cannot be redistributed, or require purchase or human approval.
```

Comment:
```suggestion
For each dataset used, authors are expected to provide a full data citation. This should include a URL for publicly available datasets, or clear instructions on how to compile or request access for datasets that cannot be redistributed, or require purchase or human approval.
```

Reply:
should we link to AEA guidance on citation? could say something like "See guidance from a consortium of data editors at top economics journals here: https://social-science-data-editors.github.io/guidance/Guidance/addtl-data-citation-guidance.html"